javascript/src_js/wrapAsm.d.ts

```diff
@@ -135,7 +135,7 @@ export type Node = {
   setHeightPercent(height: number): void,
   setJustifyContent(justifyContent: Justify): void,
   setGap(gutter: Gutter, gapLength: number): Value,
-  setMargin(edge: Edge, margin: number): void,
+  setMargin(edge: Edge, margin: number | string): void,
```

Comment:
This matches the other types here (converted from previous flowtype typings), but we could also replace this + others with a template literal type for better safety (added in TS 4.1, which is the oldest supported by DefinitelyTyped).

```suggestion
  setMargin(edge: Edge, margin: number | 'auto'  | `${number}%`): void,
```